### ISSUE TITLE
Set tilt angles by pasting from spreadsheet

### ISSUE THREAD

**Issue by ElliotPadgett** [OpenChemistry/tomviz#1421]
Sometimes the angles associated with a tomography tilt series are not just equally spaced, so users may want to enter custom angles.  The current UI supports this using the "Set Individually" tab in the "Set Tilt Angles" transform.  The most convenient way to use this would be to paste in the angles after copying from a spreadsheet, or something similar.  The entry form looks like a spreadsheet, so it is tempting for the user to try this, but when I try the UI does not respond.  If this could be implemented that would be very nice, and would save users from having to type ~100+ angles manually.

**Comment by mathturtle** [comment]
This should work already (see #1318).  I was unable to reproduce the error that was reported on that branch, but maybe you are seeing it?

**Comment by cryos** [comment]
You should be able to save the angles to a file, and load that file. Pasting should work as Shawn said, if not we should look into what the issue might be.

**Comment by ElliotPadgett** [comment]
I don't get an error message.  The UI just does nothing when I try to paste.

I'm not sure how I would go about loading an angles file.

**Comment by ElliotPadgett** [comment]
I am using Office for Mac 2011.

**Comment by ElliotPadgett** [comment]
Pasting from a Google spreadsheet doesn't work either.

**Comment by mathturtle** [comment]
This may be a Mac thing.  I'll try it on my Mac.

**Comment by mathturtle** [comment]
So I don't have Office for Mac, but I just tried Google spreadsheet and it worked (copying from a column of values in the spreadsheet).  I did find a bug on Linux when copying from LibreOffice Calc, but that shouldn't have led to a hung application, just an error message.

There should be a button on the dialog ("Load from Text File") to load the tilts from a file.  If that isn't there, then what version of tomviz are you using?
